source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -81,31 +105,34 @@ parse error and MUST NOT do DNS resolution or contact hosts.
 It is an error to specify more than one host name in a connection string with the `mongodb+srv` protocol, and the driver
 MUST raise a parse error and MUST NOT do DNS resolution or contact hosts.
 
-A driver MUST verify that in addition to the `{hostname}`, the `{domainname}` consists of at least two parts: the domain
-name, and a TLD. Drivers MUST raise an error and MUST NOT contact the DNS server to obtain SRV (or TXT records) if the
-full URI does not consist of at least three parts.
+Node:
 
 If `mongodb+srv` is used, a driver MUST implicitly also enable TLS. Clients can turn this off by passing `tls=false` in
 either the Connection String, or options passed in as parameters in code to the MongoClient constructor (or equivalent
 API for each driver), but not through a TXT record (discussed in a later section).
 
 #### Querying DNS
 
-In this preprocessing step, the driver will query the DNS server for SRV records on `{hostname}.{domainname}`, prefixed
-with the SRV service name and protocol. The SRV service name is provided in the `srvServiceName` URI option and defaults
-to `mongodb`. The protocol is always `tcp`. After prefixing, the URI should look like:
-`_{srvServiceName}._tcp.{hostname}.{domainname}`. This DNS query is expected to respond with one or more SRV records.
+In this preprocessing step, the driver will query the DNS server for SRV records on the hostname, prefixed with the SRV
+service name and protocol. The SRV service name is provided in the `srvServiceName` URI option and defaults to
+`mongodb`. The protocol is always `tcp`. After prefixing, the URI should look like: `_{srvServiceName}._tcp.{hostname}`.
+This DNS query is expected to respond with one or more SRV records.
 
 The priority and weight fields in returned SRV records MUST be ignored.
 
 If the DNS result returns no SRV records, or no records at all, or a DNS error happens, an error MUST be raised
 indicating that the URI could not be used to find hostnames. The error SHALL include the reason why they could not be
 found.
 
-A driver MUST verify that the host names returned through SRV records have the same parent `{domainname}`. Drivers MUST
-raise an error and MUST NOT initiate a connection to any returned host name which does not share the same
+A driver MUST verify that the host names returned through SRV records end with the original SRV's `{domainname}`.
```

Comment:
Sorry to be obnoxious, but... I think "end with" is potentially ambiguous? Consider the example above:
```
     - `{hostname}` = `cluster_1.tests.mongodb.co.uk`

       - `{subdomain}` = `cluster_1`
       - `{domainname}` = `tests.mongodb.co.uk`
 ```
In this case, the `{domainname}` is `tests.mongodb.co.uk`.

Suppose an SRV lookup for the `{hostname}` returned `cluster_1.eviltests.mongodb.co.uk`.

Technically speaking, doesn't `cluster_1.eviltests.mongodb.co.uk` "end with" `tests.mongodb.co.uk`?

Reply:
That's true, I hadn't considered it. I'm not sure what to replace this working with. I think using 'parent domain' was also confusing since we're allowing 'sibling domains' for SRVs with >3 parts (ex: `mongodb.co.uk` and `other.co.uk` are sibling domains currently allowed). What do you suggest?

Do you think it's implied that there must be a `.` separation if the returned host contains an extra domain level? It might be a bit clunky to repeat throughout the document, but I'm happy to put it in if it is a clarification!